services/spar/src/Spar/API.hs

```diff
@@ -278,31 +279,47 @@ validateNewIdP ::
   (HasCallStack, m ~ Spar) =>
   SAML.IdPMetadata ->
   TeamId ->
+  Maybe SAML.IdPId ->
```

Comment:
Could we add a haddock explaining the effect of `replaced` parameter here?

Reply:
173cc64c28b1fc784f239dd7e9c4004451158901

Reply:
I was hoping [this](https://github.com/wireapp/wire-server/commit/173cc64c28b1fc784f239dd7e9c4004451158901) would explain things?